hadoop-ozone/dev-support/checks/repro.sh

```diff
@@ -31,10 +33,5 @@ mv output.log "$REPORT_DIR"/
 REPORT_FILE="$REPORT_DIR/summary.txt"
 grep 'ERROR.*mismatch' "${REPORT_DIR}/output.log" > "${REPORT_FILE}"
 
-wc -l "${REPORT_FILE}" | awk '{ print $1 }' > "${REPORT_DIR}/failures"
-
-if [[ -s "${REPORT_FILE}" ]]; then
-   exit 1
-fi
-
-exit $rc # result of mvn
+ERROR_PATTERN="\[ERROR\]"
+source "${DIR}/_post_process.sh"
```

Comment:
how about `integration.sh`/`junit.sh` , Can they not fail due to non-test failures?

Reply:
Thanks.  `junit.sh` already does the same, but we can replace with the shared logic.